scripts/globals/spells/enhancing_song.lua

```diff
@@ -158,7 +158,7 @@ xi.spells.enhancing.calculateSongPower = function(caster, target, spell, spellId
     -- Level 75 Victory March tiers
     elseif songEffect == xi.effect.MARCH and tier == 2 and singingLvl >= 445 then
         power = 85
-        local increase = 17
+        local increase = 23
```

Comment:
~17 looks correct to me? Admittedly I'm not sure if I'm looking at the right thing. Where did you find this number/How did you come up with it?

Reply:
17 is correct without the changes to line 131; since this PR adds more tiers to march, we need to add more levels of increase here as well

Reply:
This doesn't change existing behavior of march for skills under 567

Reply:
Oh understood, the PR diff was cutting off the bottom where increase -= 1 each loop. Thanks!